build.sh

```diff
@@ -10,7 +10,7 @@ set -e
 
 rockcraft pack -v
 
-skopeo --insecure-policy \
+/snap/rockcraft/current/bin/skopeo --insecure-policy \
```

Comment:
I encountered issues with the version of skopeo available with `apt install skopeo` being too old. As rockcraft ships with skopeo, I thought we could directly use that version. Hence also removing the note from readme to install skopeo, as it can be simplified by using the skopeo shipped with rockcraft.

Reply:
that is fair but at the same time you can solve the problem with a symlink

`ln -s /snap/rockcraft/current/bin/skopeo /usr/local/bin/skopeo`

Reply:
True, I'll add this to the readme then instead and revert this change.

Reply:
reverted this change and added it to the readme